jdk/THIRD_PARTY_README

```diff
@@ -3315,4 +3315,51 @@ claims in such list are, in fact, Essential Claims.]
 
 --- end of LICENSE ---
 
+%% This notice is provided with respect to the FreeType 2 font engine
```

Comment:
We're missing a line of dashes to separate this from other licenses, as is done above.
I think we should include the full version here (v2.12.1), similarly to the freetype.md in jdk11. The version isn't included anywhere else.